app/models/volunteer_op.rb

```diff
@@ -34,4 +42,16 @@ def self.contains_description?(text)
   def self.contains_title?(text)
     arel_table[:title].matches(text)
   end
+  
+  def address_complete?
+    address.present? && postcode.present?
+  end 
+  
+  private
+  
+  def clear_lat_lng
+    return if address_complete?
+    self.longitude = nil
+    self.latitude = nil
+  end
```

Comment:
If the address is complete, when validating a volunteer op, we geocode and then wipe the lat long? Does the volunteer op never have a lat long otherwise?